docs/interactions/Slash_Commands.md

```diff
@@ -0,0 +1,837 @@
+
+An Interaction includes the `data` that the user sent in the command, as well as some metadata. It looks like this:
+
+```js
+{
+    "type": 2,
+    "token": "A_UNIQUE_TOKEN",
+    "member": { 
+        "user": {
+            "id": 53908232506183680,
+            "username": "Mason",
+            "avatar": "a_d5efa99b3eeaa7dd43acca82f5692432",
+            "discriminator": "1337",
+            "public_flags": 131141
+        },
+        "roles": ["539082325061836999"],
+        "premium_since": null,
+        "permissions": "2147483647",
+        "pending": false,
+        "nick": null,
+        "mute": false,
+        "joined_at": "2017-03-13T19:19:14.040000+00:00",
+        "is_pending": false,
+        "deaf": false 
+    },
+    "id": "786008729715212338",
+    "guild_id": "290926798626357999",
+    "data": { 
+        "options": [{
+            "name": "cardname",
+            "value": "The Gitrog Monster"
+        }],
+        "name": "cardsearch",
+        "id": "771825006014889984" 
+    },
+    "channel_id": "645027906669510667" 
+}
+```
+
+An explanation of all the fields can be found in our [data models](#DOCS_INTERACTIONS_SLASH_COMMANDS/data-models-and-types-interaction).
+
+Now that you've gotten the data from the user, it's time to respond to them.
+
+## Responding to an Interaction
+
+Interactions--both receiving and responding--are webhooks under the hood. So responding to an Interaction is just like sending a webhook request!
+
+When responding to an interaction received **via webhook**, your server can simply respond to the received `POST` request. You'll want to respond with a `200` status code (if everything went well), as well as specifying a `type` and `data`, which is an object with parameters the same as [Execute Webhook](#DOCS_RESOURCES_WEBHOOK/execute-webhook):
+
+```py
+@app.route('/', methods=['POST'])
+def my_command():
+    if request.json["type"] == 1:        
+        return jsonify({
+            "type": 1
+        })
+    
+    else:
+        return jsonify({
+            "type": 4,
+            "data": {
+                "tts": False,
+                "content": "Congrats on sending your command!",
+                "embeds": [],
+                "allowed_mentions": []
+            }
+        })
+```
+
+If you are receiving Interactions over the gateway, you will **also need to respond via HTTP**. Responses to Interactions **are not sent as commands over the gateway**.
+
+To respond to a gateway Interaction, make a `POST` request like this. `interaction_id` is the unique id of that individual Interaction from the receieved payload. `interaction_token` is the unique token for that interaction from the received payload. **This endpoint is only valid for Interactions received over the gateway. Otherwise, respond to the `POST` request.**
```

Comment:
```suggestion
To respond to a gateway Interaction, make a `POST` request like this. `interaction_id` is the unique id of that individual Interaction from the received payload. `interaction_token` is the unique token for that interaction from the received payload. **This endpoint is only valid for Interactions received over the gateway. Otherwise, respond to the `POST` request.**
```